### PR TITLE
[1LP][RFR] Avoid flash check with bz

### PR DESCRIPTION
**Pull request by digitronik**
Signed-off-by: Nikhil Dhandre <ndhandre@redhat.com>

Purpose or Intent
=================

Just avoid flash check as it lands on wrong page. As we are getting proper request no need to block tests.

{{pytest: cfme/tests/automate/custom_button/test_service_objects.py}}

### CODE REVIEW

**Comment by sshveta** [comment]
Please address PRT failures @digitronik .
Few failures and one Error .

    E           assert 20 == (20 + 1)

cfme/tests/automate/custom_button/test_cloud_objects.py:453: AssertionError

File: cfme/tests/automate/custom_button/test_cloud_objects.py [hunk 1]
```diff
@@ -180,6 +180,8 @@ def test_custom_button_dialog_cloud_obj(appliance, dialog, request, setup_objs,
         1555331
         1574403
         1640592
+        1710350
```
Comment (mshriver) [inline cfme/tests/automate/custom_button/test_cloud_objects.py]: 100 extra points if you add all of these BZ's to `automates` meta marker for the test case ... :)

Comment (digitronik) [reply cfme/tests/automate/custom_button/test_cloud_objects.py]: Added :). I think i need to add card as i collected mostly all BZ related to custom button from start.

**Comment by digitronik** [comment]
@sshveta 
> ```
> E           assert 20 == (20 + 1)
> ```
> 
> cfme/tests/automate/custom_button/test_cloud_objects.py:453: AssertionError

Avoid this... runs showing pass need to investigate. I will raise separate PR here i'm just blocking and trying to clean run :D.

**Comment by digitronik** [comment]
@mshriver added `automate` method. Please have a look so I will get latest good result for  5-7 test.

**Comment by jawatts** [comment]
Restarting PR since https://github.com/ManageIQ/integration_tests/pull/9114# was merged. But otherwise LGTM

**Comment by jawatts** [comment]
Failures in PRT not due to this change

**Pull request merged**